### ISSUE TITLE
Getting Issues with "not" search attributes

### ISSUE THREAD

**Issue by circlesmiler** [jdalrymple/gitbeaker#1158]
**Description**
I searched a lot, but I couldn't find any help on my problem:

I try to get issues, that do NOT have special labels. I tried something like this:

```
const p1 = gitlab.Issues.all({
         groupId: GROUP_ID,
         state: "opened",
         scope: "all",
         not: {
            labels: [
               "prio::Low"
            ]
         },
         order_by: "created_at",
         sort: "asc"
      }).then((issuesJSON: any) => {
         const issues = Issue.fromJSONAsArray(issuesJSON);
         prioNeededIssues.push(...issues);
      });
```
I think that this should match the Gitlab API description:
![image](https://user-images.githubusercontent.com/22213841/93776506-e4266d80-fc23-11ea-864e-7f6bc5dd52fb.png)

But I always got a `404` error, because of following URL that was sent to my Gitlab instance:
```
https://gitlab.company.net/api/v4/groups/2/issues?not=%5Bobject+Object%5D&order_by=created_at&scope=all&sort=asc&state=opened
```

Is this a BUG or did I something wrong?

**Comment by jdalrymple** [comment]
Definitely a bug. I'm not handing the input object for not correct, as you can see from the url that's posted, it has: "%5Bobject+Object"

It wouldnt be a super hard thing to fix!

**Comment by circlesmiler** [comment]
> Definitely a bug. I'm not handing the input object for not correct, as you can see from the url that's posted, it has: "%5Bobject+Object"
> 
> It wouldnt be a super hard thing to fix!

Ok, that sounds good. Is there a chance to have a fix in the next days?

**Comment by jdalrymple** [comment]
Currently a bit backed up with some of the bug fixes tbh. Ill try my best!

**Comment by jdalrymple** [comment]
:rocket: Issue was released in [`24.2.0`](https://github.com/jdalrymple/gitbeaker/releases/tag/24.2.0) :rocket:
